CMakeLists.txt

```diff
@@ -601,3 +601,29 @@ if(WITH_TESTS)
     enable_testing ()
     add_subdirectory (tests)
 endif(WITH_TESTS)
+
+
+#building package using CPack
+include(InstallRequiredSystemLibraries)
+
+
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Program used to map keyboard keys and mouse controls to a gamepad.")
+set(CPACK_PACKAGE_VENDOR "juliagoda")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
```

Comment:
There is small problem with license.
In installation window describes license as "Proprietary"
I don't know how to fix it.

Reply:
I compared app's license to this one from the site: https://www.gnu.org/licenses/gpl-3.0.txt and I found minor differences, maybe that's the problem, maybe not... if it's not the problem we will think again. I just updated LICENSE file

Reply:
I have checked it.
It is not the problem, installer still informs about Proprietary License.

Reply:
I am not sure about this, but I've found example on  https://accserv.lepp.cornell.edu/svn/packages/hdf5/CMakeInstallation.cmake, and there is:

```
#  If CPACK_RESOURCE_FILE_LICENSE has an .rtf extension it is used as-is.
#
#  If CPACK_RESOURCE_FILE_LICENSE has an .txt extension it is implicitly
#  converted to RTF by the WiX Generator.
#  The expected encoding of the .txt file is UTF-8.
#
#  With CPACK_WIX_LICENSE_RTF you can override the license file used by the
#  WiX Generator in case CPACK_RESOURCE_FILE_LICENSE is in an unsupported
#  format or the .txt -> .rtf conversion does not work as expected
```

so maybe that's the problem

Reply:
This example shows generating license for windows (see `if (WIN32)` above) and in case of compiling for linux license is linked in the same way as in our CMakeLists.txt.
Because of lack of ideas I have opened question on stackoverflow.com. Maybe I will soon get an answer. [link](https://stackoverflow.com/questions/61341305/cpack-deb-set-license-type)

For now I woud leave it as it is, because it is minor problem.
I would open an issue to solve later.

Reply:
It seems to ubuntu-related [issue](https://bugs.launchpad.net/ubuntu/+source/software-center/+bug/435183)

Reply:
Ok thanks